pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
This doesn't look like it should
```suggestion
    return x - y
```